tls/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - TLS
 
+## 1.3.0 / 2019-12-02
+
+* [Added] Upgrade cryptography to 2.8. See [#5047](https://github.com/DataDog/integrations-core/pull/5047).
```

Comment:
Question — in general, do dependency upgrades cause a `Fixed` or an `Added`? I've seen both being used so far. Is there a rule of thumb for deciding between the two, or is it just not standardized yet?